assets/css/quiz-style.css

```diff
@@ -0,0 +1,90 @@
+.quiz-container {
+	display: flex;
+	flex-direction: column;
+}
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Hover effect color 'red' may not align with the theme's color palette.

Consider using a color from the defined palette for consistency.

```suggestion
.quiz-container .quiz-body .quiz-options button:hover {
	background: #ff0000; /* Assuming #ff0000 is part of your defined palette */
}
```